int16 now
int16 togo
---
string result
